.github/workflows/main.yaml

```diff
@@ -508,4 +508,31 @@ jobs:
     steps:
       - name: 👌 Release Created
         run: |
-          curl --header "Content-Type: application/xml" --header "Accept: application/xml" -u '${{ secrets.N8N_WEBHOOK_BASIC_AUTH_USERNAME }}:${{ secrets.N8N_WEBHOOK_BASIC_AUTH_PASSWORD }}' --request POST --data '<Request><Version>${{ needs.create_release.outputs.tag_name }} OSS</Version><Notes>nowCRM Open source https://github.com/nowtec/nowCRM ${{ needs.create_release.outputs.body }}</Notes></Request>' ${{ secrets.N8N_NT_RELEASE_WEBHOOK }}
+          RELEASE_NOTES="nowCRM Open source https://github.com/nowtec/nowCRM
+
+          ---
+
+          Production Update Instructions
+
+          To update your production environment to version \${{ needs.create_release.outputs.tag_name }}:
+
+          1. Set the VERSION environment variable (or place it in a .env file):
+             export VERSION=\${{ needs.create_release.outputs.tag_name }}
+
+          2. Pull the new images:
+             VERSION=\${{ needs.create_release.outputs.tag_name }} docker compose -f docker-compose.prod.yaml pull
+
+          3. Restart services with the new images:
+             VERSION=\${{ needs.create_release.outputs.tag_name }} docker compose -f docker-compose.prod.yaml up -d
+
+          Note: If you have not set up production deployment yet, see the Production Deployment section in the README at https://github.com/nowtec/nowCRM for complete setup instructions.
+
+          ---
+
+          \${{ needs.create_release.outputs.body }}"
+
+          curl --header "Content-Type: application/xml" --header "Accept: application/xml" \
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `curl-auth-user` in commit 6a34b5895f1b279d458823034723cf76a60d41ed.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo 6a34b5895f1b279d458823034723cf76a60d41ed:.github/workflows/main.yaml:curl-auth-user:534 >> .gitleaksignore
```